LookingMenu/Scenes/Home Screen/Cell/RecipeRandomCell.swift

```diff
@@ -0,0 +1,133 @@
+import UIKit
+
+private enum ConstantRecipeRandomCelll {
+    case radiusView
```

Comment:
```suggestion
    static let radiusView: CGFloat = 30
```
nếu enum này chỉ toàn `CGFloat` thì có thể conform luôn `CGFloat` và dùng `rawValue`